kafka/producer.go

```diff
@@ -91,6 +102,7 @@ func (ams *asyncMessageSink) doPublishMessages(ctx context.Context, producer sar
 
 			message.Metadata = m
 			input <- message
+			ams.debugger.Logf("substrate : sent to kafka : %s\n", m)
```

Comment:
Same here for message send being in a select. Can raise a PR for that if you don't want to do that in this one...